.github/workflows/spell-checker.yml

```
@@ -0,0 +1,14 @@
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  yaml-format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Spelling
+        uses: check-spelling/check-spelling@v0.0.24
+        continue-on-error: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)